.github/workflows/nginx-build.yaml

```diff
@@ -10,16 +10,11 @@ defaults:
   run:
     working-directory: ./
 jobs:
-  deploy:
-    if: github.repository_owner == 'bcgov'
-    runs-on: ubuntu-latest
-    steps:
-      - name: Image Build for Nginx Image
-        uses: redhat-developer/openshift-actions@v1.1
-        with:
-          version: "latest"
-          openshift_server_url: ${{ secrets.OpenShiftServerURL }}
-          parameters: '{"apitoken": "${{ secrets.OpenShiftToken }}", "acceptUntrustedCerts": "true"}'
-          cmd: |
-            oc project 23576a-tools
-            oc start-build wordpress-nginx-run --follow
\ No newline at end of file
+  oc:
+    uses: bcgov/wordpress/.github/workflows/oc-setup.yaml@main
+    with:
+      image: wordpress-nginx-run
+    secrets:
```

Comment:
i wonder if this is really required, for each build, why can't the secrets be simply used in the oc-setup.  The only thing that i think needs to be passed is the image.

Reply:
See my comment above: it may be possible to relocate these secrets, but it may take a bit more work than it appears to get it there.